src/extension/content-script/batteries/VimeoVideo.ts

```diff
@@ -16,6 +16,7 @@ const battery = (): Battery | void => {
   const text = videoDescription.textContent || "";
   let match;
   let recipient;
+  const contentUri = document.location.toString();
```

Comment:
could you remove these changes for now (vimeo, medium, youtube channel)? I don't think document.location is correct (it's the full URL and can contain strange query parameters etc.). We haven't aligned on how to pass metadata yet, so I think we should focus on just one battery - the youtube video one.